lib/widgets/explore/ad_item.dart

```diff
@@ -0,0 +1,50 @@
+import 'package:dibbs/design/my_attributes.dart';
+import 'package:flutter/material.dart';
+
+class AdItem extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return GridTile(
+      child: GestureDetector(
+        child: Padding(
+          padding: MyAttributes.adItemPadding,
+          child: Column(
+            children: [
+              Container(
+                child: AspectRatio(
+                  aspectRatio: 4 / 3,
+                  child: Image(
+                    // TODO: Change this to actual ad picture
+                    image: NetworkImage(
+                        'https://www.dictionary.com/e/wp-content/uploads/2018/07/yeet2.jpg'),
```

Comment:
Nice

Reply:
Try with https://upload.wikimedia.org/wikipedia/commons/4/44/3-4_ratio_mobile_wallpaper_example.jpg to see that the image has the correct format.